niworkflows/viz/plots.py

```diff
@@ -150,16 +150,16 @@ def plot_carpet(
     Parameters
     ----------
 
-        func : string
-            Path to NIfTI or CIFTI BOLD image
+        func : string or nibabel-image object
+            Path to NIfTI or CIFTI BOLD image, or a nibabel-image object
         atlaslabels: ndarray, optional
             A 3D array of integer labels from an atlas, resampled into ``img`` space.
             Required if ``func`` is a NIfTI image.
         detrend : boolean, optional
             Detrend and standardize the data prior to plotting.
         nskip : int, optional
             Number of volumes at the beginning of the scan marked as nonsteady state.
-            Not used.
+            Only used by volumetric NIfTI.
```

Comment:
@mgxd I'm realizing of an important issue with CIFTIs - are we dropping nonsteady states before sampling?